install.php

```diff
@@ -10,7 +10,7 @@ function extension_install_uptime()
     $commonObject -> sqlQuery("CREATE TABLE IF NOT EXISTS `uptime` (
                               `ID` INT(11) NOT NULL AUTO_INCREMENT,
                               `HARDWARE_ID` INT(11) NOT NULL,
-                              `TIME` VARCHAR(64) DEFAULT NULL,
+                              `LOG_DATE` DATETIME DEFAULT NULL,
```

Comment:
Since you are changing the table model, I'd recommend to force drop the table if it exist and then re-create it.
In your case if someone update the plugin from an older version it will still have the old table format.